### DISCUSSION TITLE
Galactic Scale 2.0 Beta

### DISCUSSION THREAD

**Discussion by innominata** [Touhma/DSP_Plugins#108] [General]
2.0.0 PATCH - Beta
Saves are not compatible.

At long last, Galactic Scale 2.0 has progressed to a point where we have run out of bugs to fix.
We are now releasing the first Beta to the public.

Installation instructions:
- Remove all previous versions
- Extract GalacticScale2b65.dll and galacticbundle to BepInEx\plugins\GalacticScale (Create the directory)
- Settings are all in-game in the settings menu.

For any help or feedback, please contact us on discord!

What Galactic Scale 2.0 is: A Framework for creating generator plugins that can completely customize everything to do with galaxy/planet generation in Dyson Sphere Program, with two different generators built in.
What can be customized?

 - Planet Sizes from 5-510 tile radius (Vanilla is 200)
 - Gas Giant Sizes from 500-5100
 - Star Counts from 1-1024+ stars
 - Planet Themes (New planet types aplenty)
 - Vein Distribution
 - Terrain Generation
 - Vegetation Distribution
 - Star location, size, luminosity etc. All exposed to configuration.
 - Binary Stars.
 - System generation. Everything is configurable. Planets can share orbits.
 - All planets can have moons, including planets that are already moons. Moonception.
 -Planet material customization. Tinting, texture replacement etc.
 -Planet shader configuration. Change ocean and atmosphere params.
 - Binary Planets.

And so much more that I've forgotten. Documentation will be coming soon!

Todo: Documentation :D

[GalacticScale2b65.zip](https://github.com/Touhma/DSP_Plugins/files/6776563/GalacticScale2b65.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/Touhma/DSP_Plugins/releases/tag/2.0.65.0'>Galactic Scale 2.0 Beta</a>.</em>

**Comment by dharizon** [comment]
great mod all around. Just have one question: 

what setting in the glactic Scale 2 generator has an impact on the dyson sphere radius? I have had multiple seeds with min range: 4000, max range: 3700 (less than min range)

Any chance to increase it to the normal 40.000 (I think) or to get a slider for min/max size? Or does any sun/planet setting interfere with it?

Thank you!

**Reply by innominata** [reply]
I believe the star radius influences it. Galaxy settings / Star scale or something like that

**Comment by XenEmpireAdmin** [comment]
The Dyson Sphere/solar sail min/max radius  is currently way WAY WAY WAAAAY under-sized, at least for O type stars (which are supposed to be the best, and capable of hosting very large spheres). Less than 1/10th their proper value, even with star size slider maxed out.

I am uncertain whether or not this issue is the same on other star types - but this is a pretty bad issue for me, since Dyson Spheres are kinda the point of Dyson Sphere program...

Would it be possible to address this issue - possibly a slider specific to Dyson/Sail orbits?

**Comment by innominata** [comment]
I've tried before and failed, but I'll try again
